.github/workflows/test-build-spack.yaml

```diff
@@ -0,0 +1,80 @@
+name: Build ESMF Using Spack
+
+on:
+  workflow_dispatch:
```

Comment:
it would be nice to add `inputs` here that allow user to specify ESMF tag to be used when dispatching manually. Default should be `develop`.

Reply:
Also need a field for the nuopc-app-protos tag. Again default to `develop`. But when running this action on a specific ESMF tag/branch, then also the nuopc-app-protros tag/branch needs to be set differently.

Reply:
I think you need to add `type: string`.